src/Common/Select.php

```diff
@@ -248,8 +248,6 @@ protected function addCol($key, $val)
      * @param string $spec The column specification: "col alias",
      * "col AS alias", or something else entirely.
      *
-     * @return null
```

Comment:
I was fixing a phpStorm lint warning that said "missing return statement". Didn't notice all the other similar occurrences in the file and throughout the code. I guess for consistency, it should `@return null` or `@return void` everywhere. What is your preference?